src/util.ts

```diff
@@ -228,19 +235,111 @@ export class Util {
         return Object.keys(matrixUsers)[0];
     }
 
-    public static async ParseCommand(action: ICommandAction, parameters: ICommandParameters, args: string[]) {
+    public static async ParseHelpMessage(
```

Comment:
"parse" help message?

Reply:
eeeeeh, `HandleHelpCommand` is prolly better, yeah